next-env.d.ts

```diff
@@ -1,5 +1,4 @@
 /// <reference types="next" />
-/// <reference types="next/types/global" />
```

Comment:
Hieronder: `// NOTE: This file should not be edited`

Reply:
@ian-emsens-shd heeft een bump gedaan van Next zelf dacht ik dus dat zal mss de reden zijn? Ik zal het svs eens lokaal testen of ik het ook heb.

Reply:
Ondertussen wel change gerevert

Reply:
Yup; bump van next is hier de boosdoener